### ISSUE TITLE
No space on disk with prophet + SparkR

### ISSUE THREAD

**Issue by shzhng** [facebookincubator/prophet#196]
Looks like may be related to https://github.com/facebookincubator/prophet/issues/162

We're trying use the R prophet package in our Spark containers. A lot of files get written out to `/tmp/RtmpABCDEF/something.so`. The symbols in the shared library look something like this:

```
00000000002a0f32 W model29f5441b73a_prophet_linear_growth_namespace::model29f5441b73a_prophet_linear_growth::~model29f5441b73a_prophet_linear_growth()
00000000002a0f32 W model29f5441b73a_prophet_linear_growth_namespace::model29f5441b73a_prophet_linear_growth::~model29f5441b73a_prophet_linear_growth()
0000000000647378 b model29f5441b73a_prophet_linear_growth_namespace::current_statement_begin__
00000000002a608a W Rcpp::class_Base::getMethods(Rcpp::XPtr<Rcpp::class_Base, Rcpp::PreserveStorage, &(void Rcpp::standard_delete_finalizer<Rcpp::class_Base>(Rcpp::class_Base*))> const&, std::string&)
00000000002a6118 W Rcpp::class_Base::has_method(std::string const&)
00000000002a64a0 W Rcpp::class_Base::getProperty(SEXPREC*, SEXPREC*)
00000000002a6180 W Rcpp::class_Base::invoke_void(SEXPREC*, SEXPREC*, SEXPREC**, int)
00000000002a6140 W Rcpp::class_Base::newInstance(SEXPREC**, int)
00000000002a655e W Rcpp::class_Base::setProperty(SEXPREC*, SEXPREC*, SEXPREC*)
00000000002a612c W Rcpp::class_Base::has_property(std::string const&)
00000000002a6214 W Rcpp::class_Base::method_names()
00000000002a62f4 W Rcpp::class_Base::methods_arity()
```

Which makes me think they are the Stan models getting cached out to disk, but never getting deleted. 

At some point, disk space gets filled up and execution fails with:

```
No space left on device 2: running command ''./r_env.zip/lib/R/bin/R' CMD config CXX 2>/dev/null' had status 1 3: In file(file, ifelse(append, "a", "w")) : cannot open file '/tmp/Rtmpob2jxp/file223185e275000.cpp': No space left on device Error : package or
namespace load failed for ‘prophet’ at org.apache.spark.api.r.RRunner.compute(RRunner.scala:108) at org.apache.spark.api.r.BaseRRDD.compute(RRDD.scala:50) at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:323) at org.apache.spark.rdd.RDD.iterator(RDD.scala:287)
```

**Comment by bletham** [comment]
This does sound like #162, which was fixed in v0.1.1. Can you check what version of Prophet you are using?
```
library(prophet)
sessionInfo()
```

**Comment by bletham** [comment]
I'm hoping this is the fixed issue from #162, but if you're still having this issue please re-open.
